pom.xml

```diff
@@ -10,7 +10,7 @@
     <packaging>jar</packaging>
 
     <properties>
-        <java.version>11</java.version>
```

Comment:
`java.version` is a spring boot thing, and this setting was at odds with the compiler version setting below. For me this was using the java 8 mode.